docs/sources/setup-grafana/configure-grafana/proxy.md

```diff
@@ -0,0 +1,50 @@
+---
+aliases:
+  - ../features/datasources/proxy/
+description: Proxy datasource connections through a secure socks proxy.
+keywords:
+  - proxy
+  - guide
+title: Proxying data source connections
+weight: 1110
+---
+
+# Proxying data source connections
+
+Grafana provides support for proxying data source connections through a Secure Socks5 Tunnel. This enables you to securely connect to data sources hosted in a different network than Grafana.
+
+In order to make use of this functionality, you will need to deploy a socks5 proxy server that supports TLS on a machine exposed to the public internet within the same network as your data source. From there, Grafana will set up a mutually trusted connection from Grafana to the Proxy, and then the Proxy can proxy the Grafana connection to your private server, without exposing your data sources to the public internet.
+
+## Known limitations
+
+- Only one socks5 proxy can be configured per Grafana instance
+- All built-in core data sources are compatible, but not all external data sources are. See the [private datasource connect docs](https://grafana.com/docs/grafana-cloud/data-configuration/configure-private-datasource-connect/#known-limitations) for an updated list of supported data sources.
+
+## Configure Grafana to use the proxy
```

Comment:
@stephaniehingtgen , this is a stacked heading, which is one heading followed immediately by another heading. We want to avoid this kind of structure. I recommend removing line 23.